File: solutions/beecrowd/1012/1012.js
const readline = require('readline')

const PI = 3.14159

const rl = readline.createInterface({
  input: process.stdin,
  terminal: false
})

;(() => {
  rl.on('line', (line) => {
    const [a, b, c] = line.split(' ').map(x => parseFloat(x))

    console.log(`TRIANGULO: ${(a * c / 2).toFixed(3)}`)
    console.log(`CIRCULO: ${(PI * c * c).toFixed(3)}`)
    console.log(`TRAPEZIO: ${(c * ((a + b) / 2)).toFixed(3)}`)
    console.log(`QUADRADO: ${(b * b).toFixed(3)}`)
    console.log(`RETANGULO: ${(a * b).toFixed(3)}`)
  })
})()

File: solutions/beecrowd/1012/1012.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        float a, b, c;
        double pi = 3.14159;

        while (input.hasNextDouble()) {
            a = input.nextFloat();
            b = input.nextFloat();
            c = input.nextFloat();

            System.out.printf("TRIANGULO: %.3f\n", a * c / 2);
            System.out.printf("CIRCULO: %.3f\n", pi * (c * c));
            System.out.printf("TRAPEZIO: %.3f\n", c * ((a + b) / 2));
            System.out.printf("QUADRADO: %.3f\n", b * b);
            System.out.printf("RETANGULO: %.3f\n", a * b);
        }
    }
}
